aspnetcore/fundamentals/host/hosted-services.md

```diff
@@ -29,7 +29,7 @@ The sample app is provided in two versions:
 
 Hosted services implement the [IHostedService](/dotnet/api/microsoft.extensions.hosting.ihostedservice) interface. The interface defines two methods for objects that are managed by the host:
 
-* [StartAsync(CancellationToken)](/dotnet/api/microsoft.extensions.hosting.ihostedservice.startasync) - Called after the server has started and [IApplicationLifetime.ApplicationStarted](/dotnet/api/microsoft.aspnetcore.hosting.iapplicationlifetime.applicationstarted) is triggered. `StartAsync` contains the logic to start the background task.
+* [StartAsync(CancellationToken)](/dotnet/api/microsoft.extensions.hosting.ihostedservice.startasync) - `StartAsync` contains the logic to start the background task. When using the [Web Host](xref:fundamentals/host/web-host), `StartAsync` is called after the server has started and [IApplicationLifetime.ApplicationStarted](/dotnet/api/microsoft.aspnetcore.hosting.iapplicationlifetime.applicationstarted) is triggered. When using the [Generic Host](xref:fundamentals/host/generic-host), `StartAsync` is called before the server has started and `ApplicationStarted` is triggered.
```

Comment:
There is no server in generic host.

Reply:
Is it better to say "host" in both cases?

Reply:
No, this is part of the host starting. Just remove the server mention from generic host.